chart/config/conditions.yaml

```diff
@@ -0,0 +1,125 @@
+# Specify the feature flag conditions applying to instance groups and
```

Comment:
I don't think this paragraph explains the purpose of the file properly.
Why is it required? Why are we doing conversions?

Reply:
Will work on better text.

Reply:
Thanks!

Reply:
See fc3d09598586e1dfc5fae74dda9bbc488fddd2bc
Note, the referenced file  `../assets/operations/memory.yaml` does exist yet.
Well, while I have something locally, it is not in a state to be committed.
In part due to the memory structure still in flux.